medals/js/functions.js

```diff
@@ -1268,3 +1277,27 @@ function randomMedal() {
     run(4, 0);
 }
 
+
+document.getElementById("vote-range-text").innerHTML = document.getElementById("vote-range").value;

```

Comment:
use textContent

Reply:
can i use innerText i use that everywhere else

Reply:
sure